att_doc_retriever_sample/py/att_doc_retriever_sample.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/local/bin/env python3
+
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+import argparse
+import inspect
+import socket
+import subprocess as sp
+import sys
+import time
+
+from os import path
+
+# import module outside of subtree
+current_dir = path.dirname(path.abspath(inspect.getfile(inspect.currentframe())))
+vsock_dir = path.join(path.dirname(path.dirname(current_dir)), 'vsock_sample/py')
+sys.path.insert(0, vsock_dir)
+vs = __import__('vsock-sample')
+
+# Binary executed
+RS_BINARY = './att_doc_retriever_sample/att_doc_retriever_sample'
+
+
+def client_handler(args):
```

Comment:
This is sort of the opposite from what I would expect: The server runs in the enclave and waits for connections, when it gets a connection it sends the output of the command to the client running in the instance.